capstone-app/src/app/modules/history/wiki-media/wiki-media.component.html

```diff
@@ -1,4 +1,4 @@
 <p *ngIf="!loading">
-  Hello
   {{wikiResult.parse.title}}
+  <p id='body'>
```

Comment:
What's happening here? It looks like there's a p tag with no close and no content. Is this supposed to be here?

Reply:
When I try to add a closing p tag it throws an error. It's very odd. I use the id to set the html from my query to the pg.

Reply:
We shouldn't have a p tag inside of a p tag anyway, nor should we manually modify the dom using vanilla javascript. I think this should be:

```
<div *ngIf="!loading">
  <h1 class="history-title">{{wikiResult.parse.title}}</h1>
  <p class="history-body">{{body}}<p>
</div>

Reply:
This is the most important thing to fix. After this is done: LGTM!

Reply:
p tags are not allowed to be nested in p tags.
Consider using a div with a p inside or a div with a div inside
https://www.w3.org/TR/html401/struct/text.html#h-9.3.1

Reply:
But how do I set the inner html of body?